package.json

```
@@ -10,12 +10,12 @@
     "react-dom": "^18.2.0",
     "react-redux": "^8.0.5",
     "react-router-dom": "^6.8.1",
-    "react-scripts": "^2.1.3",
+    "react-scripts": "^5.0.1",
     "redux": "^4.2.1",
     "web-vitals": "^2.1.4"
   },
```

Best practice to provide engines
```suggestion
  },
  "engines": {
    "node": "16.x",
    "npm": "8.19.x",
    "yarn": "1.22.x"
  },
```

---

Added That.. Thanks